src/soopervisor/cli.py

```diff
@@ -2,15 +2,19 @@
 import yaml
 import click
 
-from soopervisor import __version__
+from soopervisor import __version__ as ver
```

Comment:
```suggestion
from soopervisor import __version__
```

no need to rename here